src/osgUtil/MeshOptimizers.cpp

```diff
@@ -498,7 +498,9 @@ float findVertexScore (Vertex& vert)
         return -1.0f;
     }
     float score = 0.0f;
-    int cachePosition = vert.cachePosition;
```

Comment:
It looks like LineSegmentIntersector.cpp is missing from the diff. It does appear to have changed drastically between 3.4 and 3.6 and there are other significant changes that we made in 3.4 makOsg. I am not sure which version or combinations of the versions we should use. Probably worth a defect in ontime to track the difference and continue to use the 3.6 version for now.

Reply:
This is one case where the merge wasn't feasible.   The changes looked significant enough that I opted for the 3.6 versions.   I can open an OnTime issue in case issues pop up.

Reply:
Tracked as OnTime issue 5839 (subitem of 5327)